src/index.html

```diff
@@ -4,10 +4,242 @@
+            </p>
+            <strong class="about__paragraph">
+            We will never be late!<br>
+              Guaranteed secure transaction - 100%
+            </strong>
+          </section>
+        </section>
+      </article>
+      <article class="gallery" id="gallery">
+        <section class="gallery__block">
+          <h2 class="gallery__title">Gallery</h2>
+          <img src="./images/crown.svg" alt="Crown">
+        </section>
+        <section class="gallery__block">
+          <div class="gallery__images-wrapper gallery__images-wrapper--1">
+            <img
+              src="./images/photos/1.jpg"
+              alt="Marina Palms"
+              class="gallery__image"
+            >
+            <img
+              src="./images/photos/2.jpg"
+              alt="Marina Palms"
+              class="gallery__image"
+            >
+          </div>
+          <h3 class="gallery__subtitle">
+            <a
+              class="gallery__subtitle-link"
+              href="https://goo.gl/maps/vYt2Ckb1FUw2ZeVk8"
+            >
+              Marina Palms
+            </a>
+            / North Miami Beach, FL 33162
+          </h3>
+        </section>
+        <section class="gallery__block">
+          <div class="gallery__images-wrapper gallery__images-wrapper--2">
+            <img
+              src="./images/photos/3.jpg"
+              alt="Marina Palms"
+              class="gallery__image"
+            >
+            <img
+              src="./images/photos/4.jpg"
+              alt="Marina Palms"
+              class="gallery__image"
+            >
+          </div>
+          <h3 class="gallery__subtitle">
+            <a
+              class="gallery__subtitle-link"
+              href="https://goo.gl/maps/Lju5hUBzmrx1T5N3A"
+            >
+              Edition Residens
+            </a>
+            / Miami Beach, FL 33139
+          </h3>
+        </section>
+        <section class="gallery__block">
+          <div class="gallery__images-wrapper gallery__images-wrapper--3">
+            <img
+              src="./images/photos/5.jpg"
+              alt="Marina Palms"
+              class="gallery__image"
+            >
+            <img
+              src="./images/photos/6.jpg"
+              alt="Marina Palms"
+              class="gallery__image"
+            >
+          </div>
+          <h3 class="gallery__subtitle">
+            <a
+              class="gallery__subtitle-link"
+              href="https://goo.gl/maps/XVuiboUcstNhADwS6"
+            >
+              Faena House
+            </a>
+            / Miami Beach, FL 33140
+          </h3>
+        </section>
+        <form action="#" class="gallery__block">
```

Comment:
```suggestion
        <form
          action="#"
          class="gallery__block"
          method="POST"
        >
```
Forms should have methods